### ISSUE TITLE
cannot access account settings

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#487]
**[Submitted to the original trac issue database at 3.38pm, Saturday, 26th May 2007]**

Whenever I try to access my account settings at http://www.openstreetmap.org/user/Marc+Sch%FCtz/account, I get the following error:

  Application error
  
  Change this error message for exceptions thrown outside of an action (like in Dispatcher setups or broken Ruby code) in public/500.html

Maybe this is related to the %FC in the URL?

**Comment by tomhughes** [comment]
**Author: tom@compton.nu**
**[Added to the original trac issue at 1.36pm, Monday, 28th May 2007]**

What browser are you using? Only when I try it on my local test setup I find that Firefox tries to go to a URL with a UTF-8 encoded version of the character instead of an ISO-8859-1 encoded one. In other words this URL:

  http://www.openstreetmap.org/user/Marc+Sch%C3%BCtz/account

That URL works for me on my test setup - does it work for you on the live site?

**Comment by tomhughes** [comment]
**Author: schuetzm@gmx.net**
**[Added to the original trac issue at 8.39am, Saturday, 23rd June 2007]**

I'm using Firefox. The UTF8 encoded URL you give does indeed work, however I cannot save my settings, because the form has action="/user/Marc+Sch%FCtz/account". So this is probably not a client-side problem. The page itself is served with a charset of UTF-8 (according to Firefox's "page information").

**Comment by tomhughes** [comment]
**Author: tom@compton.nu**
**[Added to the original trac issue at 12.42pm, Sunday, 19th August 2007]**

I've just tried this and I still can't recreate it I'm afraid - the form seems to have the correct UTF-8 URL and submitting it works fine... Is this still going wrong for you?

**Comment by tomhughes** [comment]
**Author: schuetzm@gmx.net**
**[Added to the original trac issue at 2.02pm, Monday, 20th August 2007]**

Yes; the only difference now is that I don't even get an error message anymore, only a completely empty page.

**Comment by tomhughes** [comment]
**Author: tom@compton.nu**
**[Added to the original trac issue at 11.50pm, Monday, 20th August 2007]**

OK. I think I've solved this - your display name was actually in the database in iso-8859-1 rather than UTF-8. Presumably because it was set before the database was converted.

I couldn't reproduce it locally because when I created a new user it was being created with the name stored as UTF-8....

I've changed your database record to be in UTF-8 now so hopefully everything should work.
